identity-admin/webapp/components/Tabs.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Tab as _Tab,
+  Tabs as _Tabs,
+  TabList as _TabList,
+  TabPanel as _TabPanel,
+} from 'react-tabs';
+import styled from 'styled-components';
+
+export const Tab = styled(_Tab)`
+  margin-right: 4px;
+  border: 1px solid black;
+  padding: 4px;
+  user-select: none;
+  cursor: arrow;
+
+  &.is-selected {
+    color: white;
+    background: black;
+    border-bottom: 1px solid white;
+  }
+
+  &:focus {
+    outline: none;
+    box-shadow: 0 0 0 2px rgba(0, 0, 255, 0.5);
+  }
+`;
+
+export const Tabs = styled(_Tabs)`
```

Comment:
Really so that all the Tabs-related components can be imported from this file, avoiding the situation where we're getting one from the library and the rest from a module:

```ts
import { Tabs } from 'react-tabs';
import { Tab, TabList, TabPanel } from '../../components/Tabs';
```

Any thoughts on how this intent could be made clearer?

Reply:
Perhaps dropping the commented out style on the override? Can you do something like:

```
import { Tabs as _Tabs } from 'react-tabs';

export const Tabs = _Tabs;
```

Reply:
Good idea, that works!